src/absil/illib.fs

```diff
@@ -556,7 +556,7 @@ module String =
         while not (isNull !line) do
             yield !line
             line := reader.ReadLine()
-        if str.EndsWith("\n") then
+        if str.EndsWith("\n", StringComparison.Ordinal) then
```

Comment:
Could you add StartsWithOrdinal/EndsWithOrdinal extension methods to illib.fs?  Then it's easy to search and remove uses of StartsWith